cargo

```diff
@@ -7,6 +7,9 @@ REPO_ROOT=$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd -P)
 # shellcheck source=build-support/common.sh
 source "${REPO_ROOT}/build-support/common.sh"
 
+GIT_TAG=$(git describe --tags)
+export GIT_TAG
```

Comment:
What do you think about giving this a more distinct name? The current name is generic enough that it could come from a CI provider or similar? E.g. `PANTS_CARGO_GIT_TAG` or something (although the `PANTS_` prefix is potentially something to avoid)?

Also, does this need to sync up with the version used in Python? If so, this might be different when running from source?